Security.Platform.iOS/Checks/NoDebuggerCheck.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace AeroGear.Mobile.Security.Checks
+{
+    /// <summary>
+    /// Check if the device is running in Debug mode.
+    /// </summary>
+    public class NoDebuggerCheck : AbstractSecurityCheck
+    {
+        protected override string Name { get { return "Debugger Check"; } }
+
+        public NoDebuggerCheck()
+        {
+        }
+
+        public override SecurityCheckResult Check()
+        {
+#if DEBUG
```

Comment:
I think this should be removed

Reply:
That is the actual check. Same approach is used in ios-sdk - https://github.com/aerogear/aerogear-ios-sdk/blob/master/modules/security/checks/NonDebugCheck.swift#L21-L25